lib/msal-node/src/config/Configuration.ts

```diff
@@ -83,6 +84,8 @@ export type NodeSystemOptions = {
     networkClient?: INetworkModule;
     proxyUrl?: string;
     customAgentOptions?: http.AgentOptions | https.AgentOptions;
+    tokenRenewalOffsetSeconds?: number;
```

Comment:
This is not a system option. Why do we need this to be configurable anyway?

Reply:
It's used to compare the access token to, to see if the access token is expired. This is something that already exists in CommonClientConfiguration, but MSI does not extend CommonClientConfiguration. We could just keep it at it's default value of 300 ms and not allow the developer to optionally configure it, if you'd like.

Reply:
I believe it is a system option. Look in lib/msal-node/src/client/ClientCredentialClient.ts on line 127

Reply:
Can we not expose this for MSI please? I have no idea why it's exposed for ConfidentialClient, there are no scenarios that I know of where we'd want app developers to change this. Maybe in Public Client and SPA, but not in Confidential Client and MSI.